rest_framework_simplejwt/backends.py

```diff
@@ -50,7 +50,11 @@ def encode(self, payload):
             jwt_payload['iss'] = self.issuer
 
         token = jwt.encode(jwt_payload, self.signing_key, algorithm=self.algorithm)
-        return token.decode('utf-8')
+        if isinstance(token, bytes):
+            # For PyJWT <= 1.7.1
+            return token.decode('utf-8')
+        # For PyJWT >= 2.0.0a1
+        return token
```

Comment:
```suggestion
        return token.decode('utf-8') if isinstance(token, bytes) else token
```
just realized we could've done

Reply:
Yep. We could solve it in this way, but I think it's less readable, because it doesn't allow us to explain this a little bit weird condition.